### PR TITLE
Organize the sig-windows board

### PR DESCRIPTION
**Pull request by jsturtevant**
As part of [cleaning up the Windows tests](https://github.com/kubernetes/test-infra/issues/20542) we discussed in our sig meeting this week of creating a smaller dashboard that would be the "signal" for windows. We discussed keeping this dashboard smaller than the current https://testgrid.k8s.io/sig-windows-releases board and having criteria for how a test can be added to the new dash.

This is a proposal for adding only our `release-informing` (https://testgrid.k8s.io/sig-release-master-informing) jobs initially to a new `sig-windows-signal` dashboard so we can make these consistently green and then evaluate adding other tests (maybe presubmits or post submits?)

Going to hold for comment from for others who maintain tests: @jayunit100 @ravisantoshgudimetla @pjh @jeremyje @marosset @claudiubelu @chewong 

/hold

fyi - I dropped our 1.17 tests as it is now longer actively maintained: https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md#non-active-branch-history

### CODE REVIEW

File: config/jobs/kubernetes/sig-windows/windows-gce.yaml [hunk 1]
```diff
@@ -92,7 +92,7 @@ periodics:
   annotations:
     fork-per-release: "true"
     fork-per-release-replacements: "--extract=ci/latest -> --extract=ci/latest-{{.Version}}"
-    testgrid-dashboards: google-windows, sig-windows-gce, sig-windows-releases, sig-release-master-informing
+    testgrid-dashboards: google-windows, sig-windows-gce, sig-release-master-informing, sig-windows-signal
```
Comment (jsturtevant) [inline config/jobs/kubernetes/sig-windows/windows-gce.yaml:95]: Do we want to move the GCE configs into the ` config/jobs/kubernetes-sigs/sig-windows` folder? I think this is the folder where sig jobs are prefered to live.

File: config/testgrids/kubernetes/sig-windows/config.yaml [hunk 2]
```diff
@@ -10,6 +11,7 @@ dashboard_groups:
     - sig-windows-networking
 
 dashboards:
+- name: sig-windows-signal
```
Comment (jsturtevant) [inline config/testgrids/kubernetes/sig-windows/config.yaml:14]: I called it `signal` so we could add non informing jobs to it later if we wanted.  Not tied to the name

File: config/jobs/kubernetes-sigs/sig-windows/release-1.17-windows.yaml [hunk 3]
```diff
@@ -1,227 +0,0 @@
-presubmits:
```
Comment (chewong) [inline config/jobs/kubernetes-sigs/sig-windows/release-1.17-windows.yaml:1]: Is 1.17 deprecated now?

Comment (jsturtevant) [reply config/jobs/kubernetes-sigs/sig-windows/release-1.17-windows.yaml:1]: https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md#non-active-branch-history

Comment (chewong) [reply config/jobs/kubernetes-sigs/sig-windows/release-1.17-windows.yaml:1]: Didn't see the comment above

**Review by chewong — APPROVED** [review]
/lgtm

File: config/jobs/kubernetes/sig-windows/windows-gce.yaml (hunk 1 above)
Comment (jsturtevant) [reply config/jobs/kubernetes/sig-windows/windows-gce.yaml:95]: @jeremyje @pjh Any thoughts on if these should be consolidated?

**Pull request merged**